ROX-Filer/src/run.c

```diff
@@ -401,7 +401,7 @@ gboolean run_by_uri(const gchar *uri, gchar **errmsg)
 	gboolean retval;
 	gchar *tmp, *tmp2;
 	gchar *scheme;
-	gchar *cmd;
+	gchar *cmd = NULL;
```

Comment:
Change of line 404 is unnecessary. On the other hand, it doesn't hurt. It's left over from a previous attempt.